tests/Functional/QueryTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Digia\GraphQL\Test;
+
+use function Digia\GraphQL\graphql;
+use function Digia\GraphQL\Test\Functional\starWarsSchema;
+
+class QueryTest extends TestCase
+{
+    public function testCorrectlyIdentifiesR2D2AsTheHeroOfTheStarWarsSaga()
```

Comment:
I think it is easier to read if you add a comment above this function
// Correctly identifies R2D as the hero of the star wars saga